src/screens/CoursesScreen.tsx

```diff
@@ -27,15 +27,15 @@ const CoursesScreen: React.FC<CoursesScreenProps> = ({ navigation }) => {
     try {
       const fetchedCourses = await getAllCourses();
       setCourses(fetchedCourses ?? []);
-    } catch (error) {
-      setError((error as Error).message);
+    } catch (err) {
+      setError((err as Error).message);
```

Comment:
bruh... you really don't like "error", do you?

Reply:
actually error is already declared as a state in the upper scope, and I really hate seeing the warning message displayed on the screen :))

Reply:
Shit... my bad. Didn't see any warning for that.

Reply:
it's still gonna work right, it's just that the checker is a bit sensitive